lib/rubocop/cop/layout/hash_alignment.rb

```diff
@@ -248,10 +248,20 @@ def check_pairs(node)
           add_offences
         end
 
+        def specific_message(format)
+          messages = { KeyAlignment => 'Align the keys of a hash literal if '\
+                      'they span more than one line.',
+                       SeparatorAlignment => 'Align the separators of a hash '\
+                       'literal if they span more than one line.',
+                       TableAlignment => 'Align the keys and values of a hash '\
+                       'literal if they span more than one line.' }
```

Comment:
This is a constant. I think it would be better to treat it as such, that is, extract it as `MESSAGES` from the method.

And it would be great if you could add spaces in front of the backslashes and indent the second lines of the string literals. We do not have a cop for this yet, but see #6420. Something like this:

```ruby
        MESSAGES = {
          KeyAlignment => 'Align the keys of a hash literal ' \
            'if they span more than one line.',
```